inductiva/_cli/storage/storage_size.py

```diff
@@ -0,0 +1,18 @@
+"""Get the user's remote storage size in GB via CLI."""
+
+from inductiva import storage
+
+
+def storage_used(args):
+    """List the user's remote storage contents."""
+    del args
```

Comment:
I think pylint doesn't complain if we prefix the argument with `_` (but we need to check):

```suggestion
def storage_used(_args):
    """List the user's remote storage contents."""
    storage.get_space_used()
```

Reply:
It indeed complains about the `_`. But I will use `unused_args`.